superset/assets/stylesheets/superset.less

```diff
@@ -424,3 +428,12 @@ g.annotation-container {
     content: "\f0dd";
     color: @brand-primary;
 }
+
+.refresh-chart-overlay {
```

Comment:
We could use a more general name if we want to reuse that style for the loading spinner